<?xml version="1.0"?>
<launch>

	<param name="maxlen"                type="double" value="8.0" />
	<param name="minlen"                type="double" value="0.0" />
    <param name="angle_vertical_lines"  type="double" value="5.0" />
    <param name="odom_dron"             type="str" value="/drone/odometry" />  

 	<param name="imgTopic"              type="str" value="/camera/color/image_raw"/>
	<param name="depthTopic"            type="str" value="/camera/aligned_depth_to_color/image_raw" /> 

    <param name="real_flag"             type="bool" value="false" />
    <param name="tfPCL"                 type="str"  value="world" /> 
    <param name="tf_word_or_sensor"     type="bool" value="true" /> <!-- true world false sensor -->

<!-- params -->

    <group ns="vision_params">
        <rosparam file="$(find vision_matrice100)/cfg/cfg_params_sim.yaml" /> <!-- cambiar a _sim para simulacion y _real para real -->
    </group>
    
  <!-- <group ns="matrix_file">
        <rosparam file="$(find lidar_camera_fusion)/cfg/cfg_params.yaml" />
    </group>-->

   <node pkg="vision_matrice100" type="panel_feature_node" name="panel_feature_node" output="screen"/>

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find vision_matrice100)/rviz/vision_matrice100.rviz" />
    </group>

   <!-- <node pkg="tf" type="static_transform_publisher" name="to_drone" args="0 0 0 0 1 0 drone_link dron_link 100"/> -->

</launch>

